2019/swift-cheatsheet/readme.md

```diff
@@ -405,6 +405,18 @@ var namesArray = [String]()
 
 Remember, Swift is a strongly-typed language so you need to specify the data type being stored in the array at the time the array is created.
 
+The above method uses Swift's type-inference engine to determine that the value is an array of strings, as we haven't specified it with the type declaration. You can do this in two ways - longhand:
+
+```swift
+var namesArray: Array<String> = []
+```
+
+Or shorthand:
+
+```swift
+var namesArray: [String] = []
+```
+
```

Comment:
Insert a comment here "These are all equivalent and personal preference as to which you choose to use".